lib/utils/SchemaBuilderXSD.js

```diff
@@ -1320,12 +1337,17 @@ class SchemaBuilderXSD {
    */
   getComplexTypeByName(complexTypeName, complexTypes) {
     let complexType,
-      complexTypeNameFound;
+      complexTypeNameFound,
+      tnsNamespacePrefix = 'tns:';
+
+    if (this.tnsNamespace && typeof this.tnsNamespace.key === 'string') {
+      tnsNamespacePrefix = this.tnsNamespace.key + ':';
```

Comment:
Namespaces commonly have these properties https://github.com/postmanlabs/wsdl-to-postman/blob/cc4b8f54352007178317e2ccdc5e167a2dc1f5f0/lib/WsdlParserCommon.js#L95 i think here you could use "prefixFilter" instead of concatenate ":" to the key

Reply:
@LuisTejedaS `prefixFilter` was `xmlns:tns:` in some cases which is not how elements are referred as. So I kept `key + ":"` as value.

Reply:
ok, i get it. Thanks